web/src/components/lineage/Lineage.tsx

```diff
@@ -248,22 +249,23 @@ export class Lineage extends React.Component<LineageProps, LineageState> {
                 scaleXMax={MAX_ZOOM}
                 scaleYMin={MIN_ZOOM}
                 scaleYMax={MAX_ZOOM}
-                transformMatrix={INITIAL_TRANSFORM}
+                initialTransformMatrix={INITIAL_TRANSFORM}
               >
-                {zoom => {
-                  return (
-                    <Box position='relative'>
+                {zoom => (
+                  <div>
                       <svg
                         id={'GRAPH'}
                         width={parent.width}
                         height={parent.height}
                         style={{
                           cursor: zoom.isDragging ? 'grabbing' : 'grab'
                         }}
+                      // @ts-ignore
```

Comment:
This makes our linter fail. I'd really prefer not to add this if possible.

Reply:
Done